nebari-dev/nebari
# Yaml config sets

## Reference Issues or PRs
Related to https://github.com/nebari-dev/nebari/issues/2865



## What does this implement/fix?

_Put a `x` in the boxes that apply_


## Testing


## How to test this PR?
Run command in https://github.com/nebari-dev/nebari/pull/2876#issuecomment-2529594920 and see if result matches.  Check that nodes are in the expected order: (general, user, worker, then gpu nodes)



## Any other comments?
